pytest_fixtures/api_fixtures.py

```diff
@@ -347,3 +357,9 @@ def oscap_content_path():
     local_file = f"/tmp/{file_name}"
     ssh.download_file(settings.oscap.content_path, local_file)
     return local_file
+
+
+@pytest.fixture(scope='module')
```

Comment:
The fixture should be `session` scoped as its dealing with the preexisting template.